wiki/en/misc/1-index.html

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jamulus ‒ Play music online. With friends. For free."
+title: "Jamulus — Play music online with friends."
 lang: "en"
 permalink: /
 layout: mainhomepage
 ldjAppCategory: "Music"
-ldjSlogan: "Play music online. With friends. For free."
+ldjSlogan: "Play music online with friends."
 ldjCreator: "Volker Fischer and contributors"
-metadescription: "Jamulus is open source software which enables musicians to perform music, rehearse or just jam, all in real time over the Internet."
+metadescription: "Copylefted libre software for musicians to perform, rehearse or just jam — all in real time over the Internet."
```

Comment:
```suggestion
metadescription: "Copylefted libre (GPLv3+) software for musicians to perform, rehearse or just jam — all in real time over the Internet."
```

Reply:
This is what Google displays when one searches "Jamulus", and for that reason I am cautious about being so accurate at the cost of comprehension by a large audience ("copylefted" etc). Previously I tried: "Jamulus is open source software that lets musicians perform music, rehearse or just jam together, all in real time over the Internet." I do recommend that word "together" as real-time collaboration is a key differentiator that "real time" might not convey.

Reply:
"Open source software" is becoming common vocabulary whilst "Copylefted" and "libre" are not, so I would definitely shy away from them.  Given that the _software_ is GPL, language describing it should follow Free Software Foundation conventions in my view.

Reply:
@mcfnord
"together" added.

@pljones 
"Open source" is merely vocabulary. In the 24 years since its making it has only managed to mislead businesses into not focusing on freedom.
The common understanding of it is similarly removed from the freedoms offered, and instead becoming one of merely price.
You can ask 10 random people what "open source" means, and this will be your unfortunate result.
Translated, it even foregoes the trademark held for "open source", and that won't change either.
In the aggregate, undermining the ability to speak of software that has source code that is open is the opposite of language, as it becomes harder to accurately describe the world around us. Licenses like this exist, like the CC-BY-NC, and practices like that of Vivaldi just opening the source code exist.

Moreover, the older nomenclature is in the language of the FSF "free software", predating "open source" by 15 years.
This is however only in its currently used form.
The FSF dissuades the use of "open source" at every turn, because it undermines what they do.
They aren't good at it, nor does "free sofware" work, but that is beside the point.
In no more certain terms could "free software" have failed to convey freedoms.

The OSI and the FSF differ in the licenses they vouch for. (Like https://en.wikipedia.org/wiki/Sybase_Open_Watcom_Public_License for example.)

In today's marketplace, Jamulus is not trying to sell shareholders on taking risks and downplaying preconceived notions in 1998, and has everything to gain by distancing itself from its aftermath.
So much so that what Jamulus is already doing is actively trying to ensure those values are not only present, but ensured for the future in being copylefted in. Again, the opposite of what businesses operating on other models are trying to do with copyleft (use it to avoid that reassurance.) (Slandering projects by saying they are "open core" didn't work, and to confuse most anyone you ask, this practice can be modelled on libre software.
Functional copyleft for the freedoms offered is also an additional point that communicates something valuable.
This could be communicated better, but then that is also the reality one seeks to convey.
_Functional_ in this respect means for something that is offered over a network to be AGPLv3+, as opposed to only GPLv3+.

Competing on price and assuming the position of freeware is old news. The ecosystem has left this behind long ago, and instead invites the worst parts of it to an equal playing-field.

Reply:
That's why we wrote free and open source software licensed under the GPL. But that's much too long, I agree.

I did a lot of research concerning Free vs Open source software in the past, but I think we must face reality where not everything can be precise. Libre software is not well known enough, open source is but is not accurate, free software is FSF speech,...

It's always a tradeoff:
What will lead to user confusion? Whats known by users in practice? Do we want to be dev focused or appeal to a big audience (Jamulus is more on the user side from my experience, and I think we should not change that)?

--> Suggestion here would be ... open source (FLOSS) ... software
As this:
1. contains a well known term "open source"
2. gives more experienced users a hint at what it is
3. is not too technical
4. Although not "perfect" ok in the sense of both sides of people understand roughly what it is

Reply:
We want something a sixth grader would understand, which is: "Jamulus is free, open source software that lets musicians perform music, rehearse or just jam together, all in real time over the Internet."

Reply:
Text, and lots of it; the preferred medium of six-graders everywhere.